### DISCUSSION TITLE
Which container to use for training Nemo?

### DISCUSSION THREAD

**Discussion by yaroslavvb** [NVIDIA-NeMo/NeMo#7057] [General]
The container given in [README](https://github.com/NVIDIA/NeMo-Megatron-Launcher#53-using-autoconfigurator-to-find-the-optimal-configuration) is 
 https://ngc.nvidia.com/containers/ea-bignlp:nemofw-training

but clicking that link gives
<img width="442" alt="Screenshot 2023-07-17 at 10 25 19 PM" src="https://github.com/NVIDIA/NeMo/assets/23068/315e2d9f-2c59-42d0-a2da-55943b8bc61d">

Searching for ea-bignlp:nemofw-training, I get

<img width="601" alt="Screenshot 2023-07-17 at 10 25 37 PM" src="https://github.com/NVIDIA/NeMo/assets/23068/929a1a60-c122-4c33-9b53-a8b6fc88fea4">

I was able to install Nemo using conda, but it required editing megatron files -- https://github.com/NVIDIA/NeMo/issues/7055

**Comment by okuchaiev** [comment]
You should go here https://developer.nvidia.com/nemo-framework-open-beta and request language one there. It does require quick registration but everyone is auto approved. 
![image](https://github.com/NVIDIA/NeMo/assets/22035961/f23a758d-190c-406a-89c3-755f4951a72f)

**Comment by Syulin7** [comment]
@okuchaiev This link has expired, could you provide a new one?
